larray/core.py

```diff
@@ -3071,12 +4897,18 @@ def _axis_aggregate(self, op, axes=(), keepaxes=False, out=None, **kwargs):
         Parameters
         ----------
         op : function
-            a aggregate function with this signature:
+            An aggregate function with this signature:
             func(a, axis=None, dtype=None, out=None, keepdims=False)
         axes : tuple of axes, optional
-            each axis can be an Axis object, str or int
+            Each axis can be an Axis object, str or int
         out : LArray, optional
+            Alternative output array in which to place the result.
+            It must have the same shape as the expected output
         keepaxes : bool or scalar, optional
+            If this is set to True, the axes which are reduced are
+            left in the result as dimensions with size one.
+            With this option, the result will broadcast correctly against
```

Comment:
this last sentence is misleading. Contrary to raw numpy, it will broadcast correctly also without the option.